exec/java-exec/src/main/java/org/apache/drill/exec/coord/zk/ZKClusterCoordinator.java

```diff
@@ -252,6 +252,8 @@ public DrillbitEndpoint apply(ServiceInstance<DrillbitEndpoint> input) {
           builder.append(bit.getControlPort());
           builder.append(':');
           builder.append(bit.getDataPort());
+          builder.append(". Version - ");
+          builder.append(bit.getVersion());
```

Comment:
Actually, how is this even possible? Drill bits do not advertise their version in ZK, do they? How will we get the version of a remote Drillbit?